src/yunohost/app.py

```diff
@@ -2906,16 +2906,31 @@ def is_true(arg):
     if isinstance(arg, bool):
         return arg
     elif isinstance(arg, basestring):
-        true_list = ['yes', 'Yes', 'true', 'True']
-        for string in true_list:
-            if arg == string:
-                return True
-        return False
+        return arg.lower() in ['yes', 'true', 'on']
     else:
         logger.debug('arg should be a boolean or a string, got %r', arg)
         return True if arg else False
 
 
+def is_boolean(value):
```

Comment:
I ... uuh .. not sure to understand why this is in app.py ? I guess that's because it's similar to this `is_true` that's already there ? But defining a function here that only gets used in another file is pretty weird ... Idk, if we want to keep things clean and tidy we should put these two somewhere in `utils/`, but otherwise just having `is_boolean` in `settings.py` is fine too ?

Reply:
I'd rather avoid more refactoring. I'll move the new function to `settings.py`.

Reply:
Done.